os-java-run/added/run-java.sh

```diff
@@ -3,6 +3,11 @@
 # Fail on a single failed command
 set -eo pipefail
 
+if [ "${SCRIPT_DEBUG}" = "true" ] ; then
```

Comment:
We need to make sure this is documented in the image too, if it isn't already (i.e. image.yaml).